src/main/java/appeng/util/item/AEItemStack.java

```diff
@@ -155,6 +158,8 @@ public static IAEItemStack loadItemStackFromPacket(final ByteBuf data) throws IO
         final byte countReqMadeType = (byte) ((mask2 & 0x3));
         final long countRequestableCrafts = getPacketValue(countReqMadeType, data);
 
+        final float usedPercent = data.readFloat();
```

Comment:
I think it would be better to save the value as a long (54.87 float -> 5487 long) and use getPacketValue/putPacketValue as more digit precision is not needed and it will save up to 3 bytes per item in packets.